app/views/layouts/_application.html.erb

```diff
@@ -67,12 +67,16 @@
 
     <a href="#main-content" class="govuk-skip-link" tabindex="0">Skip to main content</a>
 
-    <%= govuk_header(service_name:, container_classes: ("govuk-width-container__wide" if wide_container_view?)) do |header|
+    <%= govuk_header(full_width_border: true, container_classes: ("govuk-width-container__wide" if wide_container_view?)) do |header|
       if user_signed_in?
         header.with_navigation_item(text: "Sign out", href: destroy_user_session_path)
       end
```

Comment:
We'll need to move the nav down to the service navigation component too.

The [format is very similar](https://govuk-components.netlify.app/components/service-navigation/#manually-building-the-service-navigation) so it's just a case of shifting this down a few lines 🙂